app/src/main/java/tech/redroma/yoching/activities/CreditsActivity.kt

```diff
@@ -0,0 +1,198 @@
+import tech.redroma.yoching.extensions.*
+
+class CreditsActivity : AppCompatActivity()
+{
+
+    private val views = Views()
+
+    override fun onCreate(savedInstanceState: Bundle?)
+    {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_credits)
+        overridePendingTransition(R.anim.enter_from_right, R.anim.exit_to_left)
+
+        views.inflate()
+    }
+
+    override fun finish()
+    {
+        super.finish()
+        overridePendingTransition(R.anim.enter_from_left, R.anim.exit_to_right)
+    }
+
+    private inner class Views
+    {
+        lateinit var yoActionBar: Toolbar
+        lateinit var yoActionBarTitle: TextView
+
+        lateinit var truePlayer: TextView
+        lateinit var truePlayerDescription: TextView
+        lateinit var truePlayerContainer: ViewGroup
+
+        lateinit var hugh: TextView
+        lateinit var hughDescription: TextView
+        lateinit var hughContainer: ViewGroup
+
+        lateinit var wellington: TextView
+        lateinit var wellingtonDescription: TextView
+        lateinit var wellingtonContainer: ViewGroup
+
+        lateinit var marc: TextView
+        lateinit var marcDescription: TextView
+        lateinit var marcContainer: ViewGroup
+
+        lateinit var brendan: TextView
+        lateinit var brendanDescription: TextView
+        lateinit var brendanContainer: ViewGroup
+
+        lateinit var maya: TextView
+        lateinit var mayaDescription: TextView
+        lateinit var mayaContainer: ViewGroup
+
+        fun inflate()
+        {
+            yoActionBar = findView(R.id.yo_action_bar)
+            yoActionBarTitle = findView(R.id.yo_action_bar_title)
+
+            truePlayer = findView(R.id.true_player)
+            truePlayerDescription = findView(R.id.true_player_description)
+            truePlayerContainer = findView(R.id.true_player_container)
+
+            hugh = findView(R.id.hugh)
+            hughDescription = findView(R.id.hugh_description)
+            hughContainer = findView(R.id.hugh_container)
+
+            wellington = findView(R.id.wellington)
+            wellingtonDescription = findView(R.id.wellington_description)
+            wellingtonContainer = findView(R.id.wellington_container)
+
+            marc = findView(R.id.marc)
+            marcDescription = findView(R.id.marc_description)
+            marcContainer = findView(R.id.marc_container)
+
+            brendan = findView(R.id.brendan)
+            brendanDescription = findView(R.id.brendan_description)
+            brendanContainer = findView(R.id.brendan_container)
+
+            maya = findView(R.id.maya)
+            mayaDescription = findView(R.id.maya_description)
+            mayaContainer = findView(R.id.maya_container)
+
+            setupUpButton()
+            setFonts()
+            setListeners()
+
+            truePlayer.post {
+                animateIn()
+            }
+        }
+
+        fun setupUpButton()
```

Comment:
Perhaps rename to `setupBackButton()`![bitmoji](https://render.bitstrips.com/v2/cpanel/9281362-288439948_4-s1-v1.png?transparent=1&palette=1&width=246)